### PR TITLE
Add python 3.7 to CI

### PR DESCRIPTION
**Pull request by OriolAbril**

### CODE REVIEW

**Comment by OriolAbril** [comment]
Ready for review

File: scripts/create_testenv.sh [hunk 1]
```diff
@@ -44,12 +45,8 @@ fi
...
 
 # Pyro install with pip is ~511MB. These binaries are ~91MB, somehow, and do not
-# break the build. The first is the Python 3.5 wheel, the second is 3.6.
-if [ "$PYTHON_VERSION" = "3.5" ]; then
-    pip --no-cache-dir install https://download.pytorch.org/whl/cpu/torch-0.4.1-cp35-cp35m-linux_x86_64.whl
-else
-    pip --no-cache-dir install https://download.pytorch.org/whl/cpu/torch-0.4.1-cp36-cp36m-linux_x86_64.whl
-fi
+# break the build. The link can be determined from the pytorch and the python version
+pip --no-cache-dir install "https://download.pytorch.org/whl/cpu/torch-${PYTORCH_VERSION}-cp${PYTHON_VERSION//./}-cp${PYTHON_VERSION//./}m-linux_x86_64.whl"
```
Comment (ahartikainen) [inline scripts/create_testenv.sh]: nice touch

**Comment by ahartikainen** [comment]
LGTM

File: scripts/create_testenv.sh (hunk 1 above)
Comment (OriolAbril) [reply scripts/create_testenv.sh]: Thanks :)

**Comment by ColCarroll** [comment]
The future is now! Thanks, @OriolAbril !

**Pull request merged**